lib/plaid/products/apex.rb

```diff
@@ -0,0 +1,49 @@
+module Plaid
+  # Public: Class used to call the Processor Token sub-product.
+  class ProcessorToken
+    def initialize(client)
+      @client = client
+    end
+
+    # Public: Creates an Apex bank account token from an access_token.
+    #
+    # Does a POST /processor/apex/processor_token/create call which can
+    # be used to generate a Apex bank account token for a given account ID.
+    #
+    # access_token - access_token to create a public token for
+    # account_id - ID of the account to create a bank account token for
+    #
+    # Returns a parsed JSON containing a Apex bank account token
+    def create(access_token, account_id)
+      payload = { access_token: access_token,
+                  account_id: account_id }
+      @client.post_with_auth('processor/apex/processor_token/create ',
```

Comment:
Is the space as the end of the path intentional? Also, is the path supposed to be prefixed with "processor"? If yes, then the logic for this belongs under our processor file: https://github.com/plaid/plaid-ruby/blob/master/lib/plaid/products/processor.rb